dht_bootstrap.go

```diff
@@ -148,7 +183,25 @@ func (dht *IpfsDHT) Bootstrap(_ context.Context) error {
 // RefreshRoutingTable tells the DHT to refresh it's routing tables.
 func (dht *IpfsDHT) RefreshRoutingTable() {
```

Comment:
A simpler API would be to make `RefreshRoutingTable()` always return the channel making it non-buffered, and in the send to channel above, skip and close the channel if no one's receiving (select + default branch). Such an API change wouldn't be breaking.

Reply:
SGTM. However:

* It's technically breaking but only if someone defined an interface for this.
* We _have_ to write to the channel even if nobody's listening. Otherwise, we could return before the caller starts listening.